images/stable-baselines/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG on
+FROM python:${on}
```

Comment:
SB works only with max python 3.7 (due to tensorflow limitations)

Reply:
Ok, I've updated it to use 3.7